### DISCUSSION TITLE
Content security policy

### DISCUSSION THREAD

**Discussion by theonlyalm** [searxng/searxng#3396] [Q&A]
I try to add a code snippet of Google analytics to my searxng instance but actually it gives my an error because if the CSP Content Security Policy

I tried to add a line in base.html file to allow Google domains in CSP but still gives an error

What's the solution?

**Comment by Sommerwiesel** [comment]
Using a privacy focussed search engine and then implementing Google Analytics has to be the most ridiculous idea I have heard this year...

> What's the solution?

DON'T. Sorry to be that plain, but please don't.

**Reply by theonlyalm** [reply]
Bruh I understand what you mean and I agree with you but I'm just testing something, so my question is about CSP not about privacy
